libr/util/file.c

```diff
@@ -181,11 +181,12 @@ R_API bool r_file_exists(const char *str) {
 	return S_IFREG == (S_IFREG & buf.st_mode);
 }
 
-R_API long r_file_proc_size(FILE *fd) {
+static inline long r_file_proc_size(FILE *fd) {
```

Comment:
rename it to `file_proc_size`, because we use `r_` usually for exposed functions.